frontend/src/lib/components/heroEmoji/HeroEmoji.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
```

Comment:
I think this component is too dynamic 😅. Basically, this is a `div` that accepts anything. Therefore, using this instead of a `div` doesn't provide much benefit, nor does it allow us to control what people place inside the `<HeroEmoji>` component. I know we want to provide dynamic components, but we should somehow limit what the users can do with the components. In this case, the name of the component is `heroEmoji`. Therefore, we should ensure the component only renders emojis.

https://unicode.org/emoji/charts/full-emoji-list.html

Reply:
It's true that the component is quite generic, but I think there's still value to it.

In order to provide the same functionality otherwise, we would have to define a global utility class containing the necessary styling – but that is against the [recommended practice](https://tailwindcss.com/docs/reusing-styles#extracting-components-and-partials) when using Tailwind. Their recommendation is to build components instead of semantic classes. And I think the `aria-hidden` default is also a good one for this.

I don't think restricting what is rendered is actually necessary, because there might be a good reason for someone building a VAA to use some other glyph than an emoji for illustration. But if you feel strongly that we should strip out other characters, we could use a prop instead of a slot and use the [Unicode class regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape) `/\P{Emoji}/u` to remove everything else and limit the content length (although that's a bit tricky because some emoji use up multiple chars).

What do you think?

Reply:
Oki, we can keep the slot and add no restrictions.